content/docs/optimizing-performance.md

```diff
@@ -1,93 +1,91 @@
 ---
 id: optimizing-performance
-title: Optimizing Performance
+title: Optimizando el Rendimiento
 permalink: docs/optimizing-performance.html
 redirect_from:
   - "docs/advanced-performance.html"
 ---
 
-Internally, React uses several clever techniques to minimize the number of costly DOM operations required to update the UI. For many applications, using React will lead to a fast user interface without doing much work to specifically optimize for performance. Nevertheless, there are several ways you can speed up your React application.
+Internamente, React utiliza diferentes técnicas inteligentes para minimizar el número de operaciones DOM costosas requeridas para actualizar la interfaz de usuario. Para muchas aplicaciones, el uso de React conllevará a una interfaz de usuario rápida sin hacer mucho trabajo para optimizar específicamente el rendimiento. Sin embargo, hay varias maneras de acelerar tu aplicación de React.
 
-## Use the Production Build {#use-the-production-build}
+## Usar la *Build* de Producción {#use-the-production-build}
```

Comment:
@DanielGarc @carburo @Darking360 @RamirezAlex what do you think about using `el build` instead of `la build`?

```suggestion
## Usar el *Build* de Producción {#use-the-production-build}
```

Reply:
I would personally use `el Build` de Producción , but I see how different Spanish speaking countries would use `la`.